cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -154,6 +154,10 @@ data:
 
   # (Optional) Allow starting unsupported ceph-csi image
   ROOK_CSI_ALLOW_UNSUPPORTED_VERSION: "false"
+
+  # (Optional) control the host mount of /etc/selinux for csi plugin pods.
```

Comment:
don't we need to document when this needs to be enabled?

Reply:
we are defaulting to true here.